README.md

```diff
@@ -3,7 +3,7 @@
  
 <img width="257" alt="Lance Logo" src="https://user-images.githubusercontent.com/917119/199353423-d3e202f7-0269-411d-8ff2-e747e419e492.png">
 
-**Blazing fast exploration and analysis of ML data using python and SQL, backed by an Apache-Arrow compatible data format**
+**Alternative to Parquet. 100x faster for random access, includes an embedded vector index and automatic versioning. Apache Arrow and DuckDB compatible.**
```

Comment:
embedded vector index sounds a bit weird. I think ppl will be like "do you mean embedding?". Since you already say "includes", let's shorten to just "vector index"?

Reply:
Ack, modified